primer3/argdefaults.py

```diff
@@ -42,5 +54,308 @@ class Primer3PyArguments:
     temp_only: int = 0
     calc_type_wrapper = 'ANY'
 
+    dmso_conc: float = 0.0
+    dmso_fact: float = 0.6
+    formamide_conc: float = 0.8
+    annealing_temp_c: float = 50.0  # 70.0 or 120 in primer3 source
```

Comment:
50 was our old default and 70/120 is the default in primer3 source? I actually don't even see this exposed on the primer3 web interface: https://primer3.ut.ee/

Reply:
Based on my inspection, it tends to get defaulted to `-10.0;` in `libprimer.c` and there's checks if its value is greater than zero in a bunch of code to e.g. `calc_and_check_oligo_features`, so maybe we should make it -10.0

Reply:
So it sounds like updating to -10C is the move?

Reply:
I did updated it to -10